R/splitOn.R

```diff
@@ -386,7 +394,7 @@ setGeneric("unsplitOn",
     # IF the object is TreeSE. add rowTree
     if( class_x == "TreeSummarizedExperiment" ){
         # Update or add old tree from the first element of list
-        if( update_rowTree ){
+        if( agglomerate.tree  ){
             ans <- addHierarchyTree(ans)
```

Comment:
I haven't seen this before, but this is a bug. It calculates new tree instead of pruning the existing one.

Reply:
https://github.com/microbiome/mia/issues/569